ui/src/components/tracks/dataset_slice_track.ts

```diff
@@ -92,6 +92,12 @@ export interface DatasetSliceTrackAttrs<T extends DatasetSchema> {
    *   width corresponds to the duration of the slice.
    * - `depth` (NUM): Depth of each event, used for vertical arrangement. Higher
    *   depth values are rendered lower down on the track.
+   * - `layer` (NUM): This layer value influences the mipmap function. Slices in
+   *   different layers will be mipmapped independency of each other, and the
+   *   buckets of higher layers will be rendered on top of lower layers.
+   *   Warning: While using layers, depth values must be kept under 16 (see
```

Comment:
We should enforce this somehow (maybe by just getting the max depth from the dataset) otherwise this will silently fail and be impossible to debug.